docs/src/create-apps/runtime-operations.md

```diff
@@ -0,0 +1,249 @@
+```
+
+{{< /codetabs >}}
+
+### Execute actions on your Node.js app
+
+{{< codetabs >}}
++++
+title=Ping your app
++++
+
+To ping your Node.js app, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+    pm2-ping:
+        role: admin 
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 ping $APP
+``` 
+
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-ping", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+<--->
++++
+title=Reload your app
++++
+
+To reload your Node.js app, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+    pm2-reload:
+        role: admin 
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 reload $APP
+``` 
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-reload", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+<--->
++++
+title=Restart your app
++++
+
+To restart your Node.js app, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+    pm2-restart:
+        role: admin 
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 restart $APP 
+``` 
+
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-restart", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+{{< /codetabs >}}
+
+### Define management commands on your Django project
+
+On a Django project, you can define management commands, for example to run a migration outside of the Django ORM.
+To do so, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+  manual-migration:
+    role: admin
+    commands:
+      start: python manage.py manual_migration
+```
+
+To trigger your runtime operation, run a cURL command similar to the following:
```

Comment:
Compare to Bigfoot Django, generating fake data:

- [the call](https://github.com/blackfireio/bigfoot-django/blob/main/README.md?plain=1#L42)
- [the command definition](https://github.com/blackfireio/bigfoot-django/blob/main/bigfoot/management/commands/generate_fake_data.py)